docs/user_guide/bugReport.rst

```diff
@@ -0,0 +1,148 @@
+in a software application that causes it to **behave unexpected**, **hang** or
+even **crash**. No matter how sharp our skills are, or rigorous our testing, a
+few bugs are bound to slip through the cracks every now and then. As much as you
+don't want them; we don't want/like them either. However, some bugs can be
+difficult to replicate, track down and eventually fix without a well written bug
+report. If you encounter one of the what you believe is a software bug, this
+guide is here to provide a step by step approach into reporting the bug on our
+`issue tracker <https://github.com/analogdevicesinc/scopy/issues>`__ .
+
+What we need is:
+   -  Brief description of the bug
+   -  Steps to reproduce the bug
+   -  Expected result
+   -  Actual result with a screenshot or screen capture
+
+Before reporting a bug
+----------------------
+
+Is it a bug? or is it a feature request? Something not working properly (a bug)
+is different than something working differently than you want. `Feature
+requests <https://github.com/analogdevicesinc/scopy/issues>`__ are tracked in
+the same place as bugs, but will not have a ''bug'' label.
+
+Some :ref:`troubleshooting steps <troubleshooting>`
+that might be worth looking at to understand if you have a potential common
+hardware problem.
+
+Scopy is continuously being improved from one version to another, (with bug
+fixes) and with new features. Ensure you are using the `latest release
+version <https://github.com/analogdevicesinc/scopy/releases/latest>`__ of scopy.
+If you can replicate the bug on that, try
+
+If the bug exists in the nightly - You will need to report a bug (sorry). First
+check to see if someone else has reported the same
+`bug. <https://github.com/analogdevicesinc/scopy/issues?q=is%3Aissue+is%3Aopen+label%3Abug>`__
+
+Description of the Bug
+----------------------
+
+The easiest/most straightforward way to describe the bug, is to let us see how
+scopy was configured when the bug occurred. From ne session to another, Scopy
+uses **.ini** files to load its previous state when it was closed. These file
+are located in:
+
+-  **Windows:** C:Users<your_username>AppDataRoamingADI Sometimes this is a
+   hidden directory, and you `unhide
+   it. <https://cybertext.wordpress.com/2012/05/29/cant-see-the-appdata-folder/>`__
+-  **Linux:** /home/<your_username>/.config/ADI
+-  **Mac:** /Users/<your_username>/.config/analog.com
+
+The **\*.ini** files are simple text files:
+   -  **Scopy.ini** : captures the state of all the instruments, and menu
+      settings, and locations of all windows.
+   -  **Preferences.ini** : controls the preferences of scopy
+
+When a bug/crash occurs, this files can be trivial in inspecting the
+configuration that caused Scopy to fail, and can be critical in getting allowing
+a developer to easily replicate the problem.
+
+In the Windows folder where the ".ini", a **crashdmp** folder will also be there
```

Comment:
This paragraph and what comes after can be removed since Scopyv2 no longer provides this type of debugging.

But Scopy now provides some crash dump logs with timestamp in case the application crashes. These are located in the same folder as ini files